andela/wg-7-horsemen
# Ft-Exercise-cache-when-deleting-muscles-152919276

**What does this PR do?**

It resets the cache when a user deletes a muscle.

**Description of Task to be completed?**

When a muscle is deleted, the cache is reset and removed from the muscle overview, descriptions etc.

**How to manually test the feature ?**

Delete a muscle form the muscle admin overview. Then go and check for the muscle deleted in muscle overviews and in the descriptions via exercises. The muscle should not be there.

**What are the relevant pivotal tracker stories?**

#152919276
